### PR TITLE
KTA-1 Fix Embedly error, Vimeo error, SimpleJson error

### PR DESCRIPTION
**Pull request by oscarmcm**
This PR fix this errors:

    Embedly missing API key.
    Vimeo https support.
    Use python json instead django simplejson.
    Readme.md typo fix.

It's now in 1.0.0 version please create a new github release and upload this to PyPI.

~~Note: Please add PyPI Badge to the readme.md file, read more [here.](https://badge.fury.io/for/py/magicembed)~~ DONE :smile:

### CODE REVIEW

File: magicembed/providers.py [hunk 1]
```diff
@@ -47,14 +51,22 @@ def render_video(self):
...
 
     def __init__(self, url, size=(640, 480)):
         super(Embedly, self).__init__(url, size)
-        self.api_url = 'http://api.embed.ly/1/oembed?url=%s&maxwidth=%s&format=json' % (url, size[0])
+        key = getattr(settings, "EMBEDLY_KEY", None)
+        if key != None:
+            try:
+                self.api_url = 'http://api.embed.ly/1/oembed?key=%s&url=%s&maxwidth=%s&format=json' % (key, url, size[0])
+            except IOError:
+                raise IOError("Please set the Embedly api key correctly")
```
Comment (fitoria) [inline magicembed/providers.py]: Cambiar a ValueError, IOError es error de entrada y salida

Comment (oscarmcm) [inline magicembed/providers.py]: But if the api key is '12345' in the **_call_api** method **urllib** return 404 don't receive something

Comment (oscarmcm) [inline magicembed/providers.py]: I just add another try in the **_call_api** method for this.

Comment (fitoria) [inline magicembed/providers.py]: de hecho, este try no hace sentido ya que solo haces concatenación de cadenas, cual es la posibilidad qeu algo salga mal?

**Pull request merged**